Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/UserPropActivity.java

```diff
@@ -298,6 +309,76 @@ else if(btn == subscribeInterceptmedia)
                 } else {
                     ttclient.doUnsubscribe(user.nUserID, Subscription.SUBSCRIBE_INTERCEPT_MEDIAFILE);
                 }
+            else if(btn == transmitVoice)
+                if (checked) {
+                    Channel chan = new Channel();
+                    ttclient.getChannel(user.nChannelID, chan);
+                    chan.transmitUsers[0][0] = user.nUserID;
```

Comment:
You need to assign the user-ID to the first value in `transmitUsers[0]` that is 0. Otherwise you end up erasing the first already in the list.

Reply:
I would suggest you create a separate method in Utils class for handling this toggling. Then input arguments to the method can be user-ID and StreamType.